src/hotspot/share/runtime/continuation.cpp

```diff
@@ -411,9 +411,8 @@ void Continuations::init() {
   Continuation::init();
 }
 
-// While virtual threads are in Preview, there are some VM mechanisms we disable if continuations aren't used
 bool Continuations::enabled() {
-  return VMContinuations && Arguments::enable_preview();
```

Comment:
With this change, I think header file `"runtime/arguments.hpp"` at line 29 can be excluded.

Reply:
> With this change, I think header file `"runtime/arguments.hpp"` at line 29 can be excluded.

Indeed, that header file is no longer needs to be included - thanks!